leobot_web_server/web_content/js/script.js

```diff
@@ -0,0 +1,47 @@
+var delta = 0;
+// Connection to ROS
+var ros = new ROSLIB.Ros({
+    url : 'ws://' + window.location.hostname + ':9090'
+});
+ros.on('connection', function() {
+    console.log('Connected to websocket server.');
+});
+ros.on('error', function(error) {
+    console.log('Error connecting to websocket server: ', error);
+});
+ros.on('close', function() {
+    console.log('Connection to websocket server closed.');
+});
+var headControl = new ROSLIB.Topic({
+    ros : ros,
+    name : '/leobot/head_position_controller/command',
+    messageType : 'std_msgs/Float64'
+});
+var listener = new ROSLIB.Topic({
+    ros : ros,
+    name : '/leobot/head_position_controller/command',
+    messageType : 'std_msgs/Float64'
+});
+listener.subscribe(function(message) {
```

Comment:
incorrect topic reading. This is command which someone is sending to controller. You need actual value of the joint. It could be read from /leobot/head_position_controller/state field process_value. Else someone could send command 10000 and you will read it and update your delta :)